### PR TITLE
Create a PR comment on PR env deploy

### PR DESCRIPTION
**Pull request by GiridharanNarayanan**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ function loadConfig() {
 
     // use a unique container for assets for each PR inside the shared storage account
     config.assetsContainerName = `${config.assetsContainerName}-${config.pullRequestId}`;
+    config.isPullRequest = true;
```
Comment (thomasf7) [inline src/index.js]: `config.isPullRequest` is already `true` here

File: action.yml [hunk 2]
```diff
@@ -11,6 +11,9 @@ inputs:
     description: "If true, this will deploy a unique instance based on the pull request information."
     required: false
     default: false
+  github-token:
+    description: "Github token to access github resources required during the workflow. For example posting a comment in the PR. Required if 'pull-request' is true."
```
Comment (thomasf7) [inline action.yml:15]: Can you update the readme to demonstrate how to use this?

File: src/index.js [hunk 3]
```diff
@@ -383,6 +385,26 @@ async function deploy(config) {
...
 
   console.log(`Successfully deployed to https://${name}.azurewebsites.net/`);
+
+  if (isPullRequest) {
+    try {
+      const token = core.getInput('github_token', { required: true });
+      const client = new github.GitHub(token);
+      const pullRequestNumber = github.context.payload.number;
+      const pullRequest = github.context.payload.pull_request;          
+      const commentBody = `Successfully deployed test environment to https://${config.name}.azurewebsites.net/`
```
Comment (thomasf7) [inline src/index.js]:
```suggestion
      const commentBody = `Successfully deployed test environment to https://${name}.azurewebsites.net/`
```

**Review by thomasf7 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (GiridharanNarayanan) [reply src/index.js]: I was using your old repo code locally which didn't have this in the config :) Will fix this.

**Review by thomasf7 — APPROVED** [review]

**Pull request merged**